examples/event_handler_rest/src/accessing_request_details.py

```diff
@@ -16,12 +16,12 @@
 @app.get("/todos")
 @tracer.capture_method
 def get_todos():
-    todo_id: str = app.current_event.get_query_string_value(name="id", default_value="")
+    todo_id: str = app.current_event.query_string_parameters["id"]
     # alternatively
     _: Optional[str] = app.current_event.query_string_parameters.get("id")
 
     # or multi-value query string parameters; ?category="red"&?category="blue"
-    _: List[str] = app.current_event.get_multi_value_query_string_values(name="category")
+    _: List[str] = app.current_event.multi_value_query_string_parameters["category"]
```

Comment:
How about using `.get()`? So we prevent the customer from having problems with `KeyError`.